data.json

```diff
@@ -155,11 +155,11 @@
   {
     "name": "Devfolio",
     "difficulty": "hard",
-    "description": "Four merged pull requests during the month of October will earn stickers and a chance at a limited edition t-shirt. Exceptional PRs will also earn an exclusive limited edition t-shirt.",
-    "reference": "https://devfolio.co/blog/hacktoberfest-2019-devfolio/",
-    "image": "https://devfolio.sgp1.digitaloceanspaces.com/ghost/2019/10/ghost_download1.jpeg",
-    "dateAdded": "2019-10-05T23:17:55.000Z",
-    "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
+    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags.(Ships only to India)",
```

Comment:
```suggestion
    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags. (Ships only to India)",
```

Reply:
@aslafy-z I have updated the description, please review.